django_irods/views.py

```diff
@@ -380,16 +379,6 @@ def rest_check_task_status(request, task_id, *args, **kwargs):
     '''
     if not task_id:
         task_id = request.POST.get('task_id')
-    result = AsyncResult(task_id)
-    if result.ready():
-        try:
-            ret_value = result.get()
-            return JsonResponse({"status": 'true',
-                                 'payload': str(ret_value)})
-        # use the Broad scope Exception to catch all exception types since this view function can be used for all tasks
-        except Exception:
-            # logging exception will log the full stack trace and prepend a line with the message str input argument
-            logger.exception('An exception is raised from task {}'.format(task_id))
-            return JsonResponse({"status": 'false'}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
-    else:
-        return JsonResponse({"status": None})
+
+    task = get_task(task_id)
```

Comment:
why not use `get_or_create_task_notification` directly here instead of defining and importing `get_task` wrapper?

Reply:
should note I think it's okay to remove the exception handling here because Celery doesn't propagate exceptions (unless we configure it to, AFAIK we haven't set [task-eager-propagates](https://docs.celeryq.dev/en/v4.4.7/userguide/configuration.html#task-eager-propagates) or anything...

Reply:
Fixed it.